publish2rs.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+import os
+import kinto_http
+import hashlib
+import pathlib
+import uuid
+
+from settings import config
+
+from constants import WEBKIT_LISTS_DIR
+
+def get_config_if_env(env_var, config_section, config_option, fallback=""):
+    """
+    Return the config value if the environment variable exists; otherwise, return the fallback.
+    """
+    if env_var in os.environ:
+        return config.get(config_section, config_option, fallback=fallback)
+    return fallback
+
+def get_file_hash(list_path):
+    with open(list_path, "rb") as f:
+        return hashlib.sha256(f.read()).hexdigest()
+
+SERVER = get_config_if_env("SERVER", "main", "remote_settings_url")
+BUCKET = config.get("main", "remote_settings_bucket")
+COLLECTION = "tracking-protection-lists-ios"
```

Comment:
this collection exists in DEV but not in STAGE/PROD. Do you plan on reusing the existing `tracking-protection-lists`?

Reply:
Yes. I will push to stage tomorrow. And I think I need to open a bug with a schema to push to prod right ?

Reply:
Thanks. Done. I opened: https://bugzilla.mozilla.org/show_bug.cgi?id=1933955